### ISSUE TITLE
Question: Enabling (pseudo) inlay hints

### ISSUE THREAD

**Issue by williamboman** [jose-elias-alvarez/typescript.nvim#6]
Hello! After switching from [nvim-lsp-ts-utils](https://github.com/jose-elias-alvarez/nvim-lsp-ts-utils) I've noticed that I'm no longer able to enable inlay hints (via virtual text). Is the intention perhaps to hold off on inlay support in typescript.nvim until proper inlays are supported in neovim? Would you consider porting it to typescript.nvim using the same approach as in nvim-lsp-ts-utils, or do you recommend I revert back to nvim-lsp-ts-utils for inlay support?

Thanks for your work 🙏!

**Comment by jose-elias-alvarez** [comment]
I started to port the inlay hints implementation from nvim-lsp-ts-utils when I first started working on this plugin, but so much of the code is not specific to `typescript-language-server`, so I think waiting for (or ideally contributing to) an upstream implementation before adding support here is the right approach. I'd recommend going back, since I don't think I'm likely to work on it or accept a PR. 

I also agree with mjlbach's comment on [this issue](https://github.com/neovim/neovim/issues/18086) that anticonceal is the way to go - I wasn't thrilled with the virtual text implementation and am not currently using the feature because of that.
